app-modules/ai/src/Filament/Pages/ManageAiIntegratedAssistantSettings.php

```diff
@@ -103,7 +103,7 @@ public function save(): void
      */
     public function getFormActions(): array
     {
-        if (! auth()->user()->can('product_admin.*.update')) {
+        if (! auth()->user()->can('saas_global_admin.*.update')) {
```

Comment:
This should be checking if they are a global admin rather than a permission. There is a helper for this. `$user->isSuperAdmin()`